### ISSUE TITLE
shell: datapusher container isn't started automatically

### ISSUE THREAD

**Issue by wardi** [datacats/datacats#308]
from a stopped state I can't run `datacats shell`

```
FATA[0000] Error response from daemon: Could not get container for datacats_datapusher_bare24_primary
```

**Comment by wardi** [comment]
@deniszgonjanin  this looks like we shouldn't be bothering to connect to datapusher from interactive_shell, right? Or if there is a reason to talk to datapusher then datapusher should be started as a supporting container, *not* when ckan starts (and stopped when ckan stops)

**Comment by deniszgonjanin** [comment]
:+1: seeing this error everywhere now, with fresh and existing datacats environments.

**Comment by JackMc** [comment]
Huh. Got an assignment to do then I'll look into this

**Comment by wardi** [comment]
fixed by #346
